LockboxXCUITests/LockboxXCUITests.swift

```diff
@@ -4,87 +4,29 @@
 
 import XCTest
 
-let emailTestAccountLogins = "test-b62feb2ed6@restmail.net"
-let passwordTestAccountLogins = "FRCuQaPm"
-
-var uid: String!
-var code: String!
-
-let firstEntryEmail = "iosmztest@gmail.com"
-
-let getEndPoint = "http://restmail.net/mail/test-b62feb2ed6"
-let postEndPoint = "https://api.accounts.firefox.com/v1/recovery_email/verify_code"
-let deleteEndPoint = "http://restmail.net/mail/test-b62feb2ed6@restmail.net"
-
 class LockboxXCUITests: BaseTestCase {
 
-    override func setUp() {
-        // First Delete the inbox
-        let restUrl = URL(string: deleteEndPoint)
-        var request = URLRequest(url: restUrl!)
-        request.httpMethod = "DELETE"
-
-        let task = URLSession.shared.dataTask(with: request) { data, response, error in
-            print("Delete")
-        }
-        task.resume()
-        super.setUp()
+    override func tearDown() {
+        navigator.goto(Screen.AccountSettingsMenu)
+        waitforExistence(app.buttons["disconnectFirefoxLockbox.button"], timeout: 3)
+        // Using taps directly because the action is intermittently failing on BB
+        app.buttons["disconnectFirefoxLockbox.button"].tap()
+        waitforExistence(app.buttons["Disconnect"], timeout: 3)
+        app.buttons["Disconnect"].tap()
+        waitforExistence(app.buttons["getStarted.button"], timeout: 30)
+        navigator.nowAt(Screen.WelcomeScreen)
     }
 
-    func test0LoginSuccessfully() {
+    func testCheckEntryDetailsView() {
         snapshot("01Welcome" + CONTENT_SIZE)
-        userState.fxaPassword = passwordTestAccountLogins
-        userState.fxaUsername = "test-b62feb2ed6@restmail.net"
-        navigator.goto(Screen.FxASigninScreenEmail)
-        waitforExistence(app.navigationBars["Lockbox.FxAView"])
-        waitforExistence(app.webViews.textFields["Email"], timeout: 10)
-        navigator.performAction(Action.FxATypeEmail)
-        waitforExistence(app.webViews.secureTextFields["Password"])
-        navigator.performAction(Action.FxATypePassword)
-        // When the account is unverified, it is necessary to wait here while finding a different workaround
-        sleep(5)
-         // Check if the account is verified and if not, verify it
-        checkIfAccountIsVerified()
-
-        if #available(iOS 12.0, *) {
-            waitforExistence(app.buttons["setupAutofill.button"])
-            navigator.performAction(Action.NotAutofillSetUpNow)
-        }
-        waitforExistence(app.buttons["finish.button"])
-        navigator.goto(Screen.LockboxMainPage)
-
-        waitforExistence(app.tables.cells.staticTexts[firstEntryEmail], timeout: 15)
+        loginFxAccount()
```

Comment:
😍  love the new descriptive method names, this is feeling like the android ui tests!